File: Python/2-locating-elements/1-By-ID.py
from selenium import webdriver

driver = webdriver.Chrome()
driver.get("https://www.selenium.dev/documentation/en/")
x = driver.find_element_by_id('the-selenium-browser-automation-project')
y = driver.find_element('id', 'the-selenium-browser-automation-project')

print(x.text, y.text)

driver.quit()

File: Javascript/2-locating-elements/1-By-ID.js
const { Builder, By } = require('selenium-webdriver');

(async function(){
    let driver = await new Builder().forBrowser('chrome').build();
    try{
        await driver.get('https://www.selenium.dev/documentation/en/');
        const x = await driver.findElement(By.id('the-selenium-browser-automation-project'));
        console.log(await x.getText());
    } finally {
        driver.quit()
    }
})()